atomic_reactor/plugins/pre_change_from_in_df.py

```diff
@@ -96,5 +99,5 @@ def run(self):
         builder.set_base_image(parent_image_ids[df_base])
         self.log.debug(
             "for base image '%s' using local image '%s', id '%s'",
-            df_base, build_base, builder.base_image
+            df_base, build_base_str, build_base
```

Comment:
When using '%s' placeholder in log message, logging module will try to convert object to string automatically by calling the object's `__str__` method (over simplified statement). Do we need the `build_base_str` at all?

Reply:
I'm also not sure why this log message used to output got `build_base` and `builder.base_image`. I can't see how they would ever not be the same thing. @sosiouxme ?